portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,89 @@
 #content {
+  font-size: 175%;
   margin-left: auto;
   margin-right: auto;
-  width: 650px;
+  width: 70%;
+}
+
+#column {
+  display: table-cell;
+  width: 23.3%;
+}
+
+#row {
+  display: table;
+  width: 100%;
+}
+
+#secret {
+  color: white;
+}
+
+#lavendar-background {
+  background-color: lavender;
+}
+
+#blue-background {
+  background-color: lightblue;
+}
+
+#pink-background {
+  background-color: lightpink;
+}
+
+h1,
+h2 {
+  background-color:Violet;
+}
+
+h3,
+h4,
+h5 {
+  font-size: x-large;
+  text-align: center;
+}
+
+#gallery2 {
+  float: unset;
+  padding: 8px;
+  width: 200px;
+}
+
+#gallery {
+  display: block;
+  float: unset;
+  margin-left: auto;
+  margin-right: auto;
+  padding: 8px;
+  width: 200px;
+}
+
+#float-left {
+  float: left;      
+  height: 198px;
+}
+
+#rectangle {
+  border: 1px solid #000;
+  height: 200px;
+  width: 70%;
 }
 
 #greeting-container {
   margin-top: 20px;
 }
+
+#bubble-container{
```

Comment:
nit: add space before {

Reply:
Thank you so much for your edits and your patience as I am learning the style! I will try to read for style more thoroughly before I submit commits in the future.

Reply:
Yep for sure! We're all learning together, sorry for the many nits.